documentation/jetty-documentation/src/main/asciidoc/programming-guide/server/sessions/session-architecture.adoc

```diff
@@ -13,18 +13,31 @@
 
 [[pg-server-session-architecture]]
 ==== Session Architecture
+==== Introduction
```

Comment:
I'd remove this line.

FYI the number of `=` is the depth of the section, so if you mean `Introduction` to be a child of `Session Architecture`, it should have 5 `=`.

Same goes for other section titles below.